.circleci/config.yml

```diff
@@ -201,7 +201,74 @@ jobs:
 
       - run: 
           name: Run coverage
-          command: yarn prepare && yarn coverage-WPM  #triggers truffle test
+          command: yarn prepare && yarn coverage-WPM  #triggers coverage for whitePaperModel tests
+
+  deploy-ready:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:10.23.0
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/mongo:3.4.4
+      # - image: trufflesuite/ganache-cli
+      #   command: ganache-cli --gasLimit 20000000 --gasPrice 20000 --defaultBalanceEther 1000000000 --allowUnlimitedContractSize true
+
+    working_directory: ~/repo
+
+    steps:
+      - run:
+        |
+          sudo apt-get install libudev-dev
+          sudo wget https://github.com/ethereum/solidity/releases/download/v0.5.16/solc-static-linux -O /usr/local/bin/solc
+          sudo chmod +x /usr/local/bin/solc
+      - checkout
+      - run:
+          name: "Pull Submodules"
+          command: |
+            git submodule init
+            git submodule update
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "compound-protocol/package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "compound-protocol/scenario/package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run: cd compound-protocol && yarn install --lock-file
```

Comment:
do we need these commands, as we now have all executed in `npm i` ?

Reply:
Yes, we don't require those commands now. 
I had commented them in the `test` job, will remove them from all other jobs as well